### PR TITLE
scan: implement latest blocks mobile version

### LINKED ISSUE #2216: Latest Blocks (Mobile)

### PR DESCRIPTION
**Pull request by tansawit**
Fixed: #2216 

## Implementation details

Implemented latest blocks module for mobile version.

## Please ensure the following requirements are met before submitting a pull request:

- [ ] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [ ] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

## If this PR doesn't need to be merged into v1.0.6 please remove this option:

- [ ] The pull request has been cherry-picked to `rc/1.0.6`

### CODE REVIEW

File: scan/src/components/LatestBlocks.re [hunk 1]
```diff
@@ -137,3 +226,8 @@ let make = (~blocksSub: ApolloHooks.Subscription.variant(_)) => {
     </Row>
   </>;
 };
+
+[@react.component]
+let make = (~blocksSub: ApolloHooks.Subscription.variant(_)) => {
```
Comment (evilpeach) [inline scan/src/components/LatestBlocks.re]: you can remove declared type.

File: scan/src/components/LatestBlocks.re [hunk 2]
```diff
@@ -83,8 +112,68 @@ let renderMiniBlockLoading = (imaginaryIndex: int, imaginaryBlockHeight: ID.Bloc
     <LoadingCensorBar width=45 height=16 />
   </div>;
 
-[@react.component]
-let make = (~blocksSub: ApolloHooks.Subscription.variant(_)) => {
+let renderMobile = (~blocksSub: ApolloHooks.Subscription.variant(_)) => {
```
Comment (evilpeach) [inline scan/src/components/LatestBlocks.re]: Remove declared type.

File: scan/src/components/LatestBlocks.re [hunk 3]
```diff
@@ -83,8 +112,68 @@ let renderMiniBlockLoading = (imaginaryIndex: int, imaginaryBlockHeight: ID.Bloc
...
+          <Text
+            value="Latest Blocks"
+            size=Text.Xxl
+            weight=Text.Bold
+            block=true
+            color=Colors.gray8
+          />
+        </div>
+        {switch (blocksSub) {
+         | Data(blocks) =>
```
Comment (evilpeach) [inline scan/src/components/LatestBlocks.re]: Change `Data(blocks)` to `ApolloHooks.Subscription.Data(blocks)`.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by pzshine — APPROVED** [review]

File: scan/src/pages/HomePage.re [hunk 4]
```diff
@@ -73,7 +75,7 @@ let make = () => {
     // <div className=Styles.skip />
     <div className=Styles.section>
       <Row alignItems=`initial wrap=true>
-        <Col> <LatestBlocks blocksSub=lastest11BlocksSub /> </Col>
+        <Col size=1.> <LatestBlocks blocksSub=lastest11BlocksSub /> </Col>
```
Comment (pzshine) [inline scan/src/pages/HomePage.re]: Can you check `size=1.` again? It breaks on desktop view.

**Review by pzshine — CHANGES_REQUESTED** [review]

**Review by pzshine — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**